src/Json/Utils.cs

```diff
@@ -0,0 +1,33 @@
+using IO = System.IO;
+using NJson = Newtonsoft.Json;
+
+namespace Silksong.DataManager.Json;
+
+// TODO(UserIsntAvailable): Make `public`?
+
+internal static class Utils
+{
+    private static readonly NJson.JsonSerializerSettings _settings = new()
+    {
+        ContractResolver = ShouldSerializeContractResolver.Instance,
+        TypeNameHandling = NJson.TypeNameHandling.Auto,
+        ObjectCreationHandling = NJson.ObjectCreationHandling.Replace,
+        Converters = [new Vector2Converter(), new Vector3Converter()],
```

Comment:
Per discussion on a previous PR I don't think it's common enough that people actually put vectors in data that we need to vend these

Reply:
Removed!